src/StaticApi/Menus.php

```diff
@@ -47,7 +47,7 @@ public static function get_sub_menu_items( $menu_items, $parent_id ) {
 		$items = [];
 
 		foreach ( $menu_items as $menu_item ) {
-			if ( $parent_id == $menu_item->menu_item_parent ) {
+			if ( $parent_id === $menu_item->menu_item_parent ) {
```

Comment:
Did you test this @mitogh ?  If I remember rightly I had to use == because one is a string and the other an int.